src/components/Delegates/Delegations/DelegationsContainer.tsx

```diff
@@ -83,18 +80,10 @@ function DelegationsContainer({
             <Table className="min-w-full">
               <TableHeader className="text-xs text-secondary sticky top-0 bg-white z-10">
```

Comment:
maybe we could do `bg-neutral` here, sorry to be such a pest with these color changes